images/buildkit/main.tf

```diff
@@ -58,19 +59,20 @@ module "rootless" {
   main_package      = "buildkitd"
 }
 
-module "test-latest" {
+module "test-rootless" {
   source = "./tests"
-  digest = module.latest-root.dev_ref
+  digest = module.rootless.dev_ref
+  script = "build-rootless.sh"
```

Comment:
I think we probably want this to use `${path.module}/tests/build-rootless.sh`

Reply:
Done.